zerver/actions/scheduled_messages.py

```diff
@@ -178,7 +251,6 @@ def edit_scheduled_message(
         scheduled_message_object.save()
 
     notify_update_scheduled_message(sender, scheduled_message_object)
-    return scheduled_message_id
```

Comment:
Noted that the endpoints for editing messages and drafts don't return anything other that the simple success, so updated this accordingly.